### PR TITLE
[1.3 STABLE ONLY] Unbind `togglePaneZoom`

### PR DESCRIPTION
**Pull request by carlos-zamora**
## Summary of the Pull Request

Disable `togglePaneZoom` as a binding. Remove it from defaults.json and the schema.

## References
#7252 - re-enable this binding when this blocking bug is resolved

## Validation Steps Performed
Successful deployment

### CODE REVIEW

**Review by PankajBhojwani — APPROVED** [review]

File: src/cascadia/TerminalApp/ActionAndArgs.cpp [hunk 1]
```diff
@@ -22,7 +22,7 @@ static constexpr std::string_view NewTabKey{ "newTab" };
 static constexpr std::string_view NewWindowKey{ "newWindow" };
 static constexpr std::string_view NextTabKey{ "nextTab" };
 static constexpr std::string_view OpenNewTabDropdownKey{ "openNewTabDropdown" };
-static constexpr std::string_view OpenSettingsKey{ "openSettings" }; // TODO GH#2557: Add args for OpenSettings
+static constexpr std::string_view OpenSettingsKey{ "openSettings" };
```
Comment (DHowett) [inline src/cascadia/TerminalApp/ActionAndArgs.cpp:25]: I'd prefer we not make unrelated changes in this PR.

File: src/cascadia/TerminalApp/ActionAndArgs.cpp [hunk 2]
```diff
@@ -100,7 +100,7 @@ namespace winrt::TerminalApp::implementation
         { ToggleCommandPaletteKey, ShortcutAction::ToggleCommandPalette },
         { ToggleFocusModeKey, ShortcutAction::ToggleFocusMode },
         { ToggleFullscreenKey, ShortcutAction::ToggleFullscreen },
-        { TogglePaneZoomKey, ShortcutAction::TogglePaneZoom },
+        // { TogglePaneZoomKey, ShortcutAction::TogglePaneZoom }, // TODO GH#7252: Re-enable pane zooming
```
Comment (DHowett) [inline src/cascadia/TerminalApp/ActionAndArgs.cpp:103]: i believe the most minimal change _only_ requires us to comment out this line. Can you validate?

I'd prefer a +1/-1 line change to this file rather than a +4/-4.

File: src/cascadia/TerminalApp/defaults.json [hunk 3]
```diff
@@ -327,8 +327,7 @@
         { "command": { "action": "moveFocus", "direction": "down" }, "keys": "alt+down" },
         { "command": { "action": "moveFocus", "direction": "left" }, "keys": "alt+left" },
         { "command": { "action": "moveFocus", "direction": "right" }, "keys": "alt+right" },
-        { "command": { "action": "moveFocus", "direction": "up" }, "keys": "alt+up" },
-        { "command": "togglePaneZoom" },
+        { "command": { "action": "moveFocus", "direction": "up" }, "keys": "alt+up" }
```
Comment (DHowett) [inline src/cascadia/TerminalApp/defaults.json:330]: This shouldn't parse. You removed an important comma.

**Review by DHowett — CHANGES_REQUESTED** [review]
-

**Comment by carlos-zamora** [comment]
@DHowett thanks for making this stable only. Still use this in my daily workflow.
